src/elements/element.js

```diff
@@ -49,8 +49,8 @@ export default class Element {
       }
     }
 
-    this._minLength = -1;
-    this._maxLength = -1;
+    this._minLength = null;
```

Comment:
This default wasn't consistent with what the rails api returns for this attribute. (try GETing a Form from the API, and check the min_length and max_length attributes. they're all null)

Reply:
Got a second opinion that this change shouldn't cause problems, and is probably what we want (original implementation was ported from iOS and this was likely an artifact of using native types). 

I Submitted a couple of branches to fulcrum-ios and fulcrum-android that should get them started on creating a matching implementation for consistency's sake (not strictly required for these changes).